corehq/apps/reports_core/static/reports_core/js/maps.js

```diff
@@ -37,30 +52,30 @@ var maps = (function() {
             });
             var table = '<table class="table table-bordered"><%= rows %></table>';
             var template = _.template(table)({rows: rows.join('')});
-            fn.template = _.template(template);
+            privates.template = _.template(template);
```

Comment:
Yes, that's correct. It could be possible to change the columns (if you edited the report in a separate window and then re-applied the filters on this page without reloading the page). But I think that's enough of an edge case to ignore it. Reloading the page would result in the correct display.